packages/zowe-explorer/src/utils/CoreUtils.ts

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Maybe we could rename this file to `fsUtils.ts`?

Reply:
or maybe `TempFolder`? I am more a fan of defining classes with static members for these kind of utilities. That make the code easier to read and refactor.

Reply:
`TempFolder` works in my mind, too. I originally thought maybe the `cleanDir` function could be used outside of the temp folder context, but looking at the code more closely, it seems we aren't using it like that.